ts/Core/Axis/Axis.ts

```diff
@@ -5899,27 +5911,29 @@ class Axis implements AxisComposition, AxisLike {
      * @function Highcharts.Axis#adjustTickAmount
      */
     public adjustTickAmount(): void {
-        var axis = this,
+        const axis = this,
             axisOptions = axis.options,
+            series = this.series,
             tickInterval = axis.tickInterval,
             tickPositions = axis.tickPositions,
             tickAmount = axis.tickAmount,
             finalTickAmt = axis.finalTickAmt,
             currentTickAmount = tickPositions && tickPositions.length,
-            threshold = pick(axis.threshold, axis.softThreshold ? 0 : null),
-            min,
-            len,
-            i;
+            threshold = pick(axis.threshold, axis.softThreshold ? 0 : null);
+
+        let min, len, i;
 
         if (axis.hasData()) {
             if (currentTickAmount < tickAmount) {
                 min = axis.min;
 
                 while (tickPositions.length < tickAmount) {
+                    const isMinHigherThanTickInterval = (min || min === 0) && (min < tickInterval);
```

Comment:
Probably should be `Lower`, not `Higher`

Reply:
Indeed setting the `x Axis.softThreshold` will resolve this issue, unfortunately, it is causing more than 100 visual changes, and breaking a few tests.